packages/web/src/js/AutoResize.ts

```diff
@@ -1,6 +1,7 @@
 import BaseComponent from './BaseComponent';
 import { EventHandler, SelectorEngine } from './dom';
 import { enableToggleAutoloader } from './utils';
+import { SpiritConfig } from './utils/Config';
```

Comment:
Add Config export to packages/web/src/js/utils/index.ts and import it here on the third row too?

Reply:
This applies to all files.

Reply:
Fixed.